engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -132,6 +133,23 @@ case class MaximalNumberValidator(maximalNumber: BigDecimal) extends ParameterVa
   )
 }
 
+case object JsonValidator extends ParameterValidator {
+
+  //Blank value should be not validate - we want to chain validators
+  override def isValid(paramName: String, value: String, label: Option[String])(implicit nodeId: NodeId): Validated[PartSubGraphCompilationError, Unit] =
+    if (StringUtils.isBlank(value) || parse(value).isRight)
```

Comment:
1. It can be tricky, but you won't get here `{"some": "json"}` but you will get `'{"some": "json"}'` (with `'`), because we transfer all values using spring expression language, especially using: https://docs.spring.io/spring/docs/4.3.10.RELEASE/spring-framework-reference/html/expressions.html#expressions-ref-literal
It is why I've mentioned that you need to do end-to-end tests that whole solution is working correctly. Maybe you can add some example class with json parameter to `SimpleTypesService`?
2. Can you return validation message in error?

Reply:
done